maintainers/guide/code-review.md

```diff
@@ -16,19 +16,23 @@ Maintainers can assign reviews to other maintainers, when appropriate. The assig
 
 ## The Code Review Process
 
-- The **author** submits a PR
+The **author** submits a PR:
+
 - Phase 0: Select **reviewers** and **approvers** for the PR
   - Choose at least two suggested **reviewers** and request their reviews on the PR
   - Choose suggested **approvers**, based on project maintainers, and list them in a comment on the PR
+
 - Phase 1: Humans review the PR
   - **Reviewers** look for general code quality, correctness, sane software engineering, style, etc.
   - Anyone in the organization can act as a **reviewer** with the _exception_ of the individual who opened the PR
   - If the code changes look good to them, a **reviewer** types :shipit: in a PR comment or review if they change their mind.
```

Comment:
Would it be easier to just type `shipit` instead of using the emoji? Personally, I would have no idea how to create that emoji. 

Once that has been added we should automatically change the label to `status/4-merge` as well.

Reply:
Yeah, doesn't matter to me either way. Chris tends to like emojis, personally, as long as the comment is clear, I'm fine.